setup.cfg

```
@@ -34,7 +34,7 @@ scripts =
   bin/use_external_components
   bin/use_external_components.cmd
 install_requires =
-    django~=2.2.0
+    django < 4.0
```

I'd still require a minimum version of Django though

```suggestion
    django>=2.2,< 4.0
```